internal/gcs/protocol.go

```diff
@@ -118,12 +117,13 @@ type requestBase struct {
 	ActivityID  guid.GUID `json:"ActivityId"`
 }
 
-func (req *requestBase) SetActivityID(id guid.GUID) {
-	req.ActivityID = id
+func (req *requestBase) Base() *requestBase {
+	return req
 }
 
 type responseBase struct {
 	Result       int32         // HResult
+	ErrorMessage string        `json:",omitempty"`
```

Comment:
Can you put the same comment here:
`// ErrorMessage is LCOW specific and is intended to be used only for ...`

Reply:
I'm going to get this added to the inbox bridge too, so the comment won't be necessary.

Reply:
Sounds good